### ISSUE TITLE
minlength attribute incorrectly not allowed on input elements

### ISSUE THREAD

**Issue by StevenMaude** [validator/validator#208]
I've tried the following example locally using the latest 16.1.1 release on GitHub and on the Nu Html Checker (16.1.6) but get `error: Attribute “minlength” not allowed on element “input” at this point` in both of them:

```html
<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
  </head>
  <body>
    <form>
      <input required name="password" type="password" minlength="10">
    </form>
  </body>
</html>
```

I think that ab941a85936a39628570df109493be25c09b7cae was intended to make this validate, but maybe I'm mistaken.

(It's documented here in the [HTML5 specification](http://www.w3.org/TR/html5/forms.html#password-state-%28type=password%29), but I also note the discussion in #164, so you may have a deliberate reason for not allowing this.)

**Comment by sideshowbarker** [comment]
Thanks for catching this. Fix pushed to https://validator.w3.org/nu/ and I’ll also get a new release out with the fix soonish.

**Comment by StevenMaude** [comment]
@sideshowbarker Thanks for the *ridiculously* quick turnaround on fixing this :smiley:

**Comment by sideshowbarker** [comment]
Cheers & thanks for taking time to report it

**Comment by pwaller** [comment]
@sideshowbarker will there be a release with this in some time? Thanks!

**Comment by sideshowbarker** [comment]
Yes, will try to get a new release out by tomorrow this time at the latest

**Comment by pwaller** [comment]
Awesome! :clap:

**Comment by sideshowbarker** [comment]
I’ve ended up delaying the release while troubleshooting https://github.com/validator/validator/issues/245 but if I don’t get that unwound sometime I’ll probably go ahead and just release and just list that as a known issue for now.

**Comment by pwaller** [comment]
Thanks for the update. I'm happy also to wait.

**Comment by pwaller** [comment]
@sideshowbarker any progress?

**Comment by sideshowbarker** [comment]
> @sideshowbarker any progress?

v16.3.3 released https://github.com/validator/validator/releases/tag/16.3.3 and should also show up on maven central soon.

Sorry for having dragged out the release date longer than planned.
